### ISSUE TITLE
"Subject of a VC" does not exist

### ISSUE THREAD

**Issue by RieksJ** [w3c/vc-data-model#957]
In the VCDM, the term 'subject' is only **defined** in relation to claims, not in relation to credentials. Since a VC is a (non-empty) collection of claims, the phrase "subject of a VC" is generally indeterminate - it only makes sense if the VC contains a single claim. 

I have witnessed many discussions in which people have assumed that every VC does have a subject, and it causes lots of confusion, for example between people that are attempting to author a paper in the context of RWOT on 'holder binding'.

It would really help if the VCDM were very clear in this respect, but it currently is not: the phrase "subject of the [verifiable] credential" appears multiple times (even in the definition of "[holder](https://www.w3.org/TR/vc-data-model/#dfn-holders)"), and it causes various people to argue that it is ok to talk about "the subject of a VC". 

I suggest to
- change the replace all occurrences of "subject of the credential" and "subject of the verifiable credential" by texts that state what actually is the case, e.g., "subject of a claim in the (verifiable) credential".
- replace the definition of 'subject' with something like "The (single) entity to which a given set of coherent data relates/pertains. In a VC, every claim has a subject (VCs themselves do not)."

**Comment by awoie** [comment]
> In the VCDM, the term 'subject' is only **defined** in relation to claims, not in relation to credentials. Since a VC is a (non-empty) collection of claims, the phrase "subject of a VC" is generally indeterminate - it only makes sense if the VC contains a single claim.
> 
> I have witnessed many discussions in which people have assumed that every VC does have a subject, and it causes lots of confusion, for example between people that are attempting to author a paper in the context of RWOT on 'holder binding'.
> 
> It would really help if the VCDM were very clear in this respect, but it currently is not: the phrase "subject of the [verifiable] credential" appears multiple times (even in the definition of "[holder](https://www.w3.org/TR/vc-data-model/#dfn-holders)"), and it causes various people to argue that it is ok to talk about "the subject of a VC".
> 
> I suggest to
> 
> * change the replace all occurrences of "subject of the credential" and "subject of the verifiable credential" by texts that state what actually is the case, e.g., "subject of a claim in the (verifiable) credential".
> * replace the definition of 'subject' with something like "The (single) entity to which a given set of coherent data relates/pertains. In a VC, every claim has a subject (VCs themselves do not)."

I agree that this text is confusing. We also had the discussion on whether without a `credentialSubject.id`, i.e. an explicit identifier, can also have an implicit subject. This is for example important for people that don't use unique identifiers for identifying the subject -> e.g., anoncreds and potentially others such as BBS+.

**Comment by RieksJ** [comment]
VCDM says that a claim without a `credentialSubject.id` is a 'bearer-claim'.

**Comment by TallTed** [comment]
> VCDM says that a claim without a `credentialSubject.id` is a 'bearer-claim'.

I would call that a bug.  Such a claim *might* be a "bearer-claim", but I do not think the bearer is the only possible implicit subject.  I can conceive of, for instance, a scenario where the VC says (in pseudo code) "the entity with value `x` for property `y` also has value `q` for property `r`" where each value of property `y` is only ever associated with a single individual.

**Comment by RieksJ** [comment]
I agree. The formal semantics for claims without a `credentialSubject.id` is that the claim is made about some (further unspecified) entity that the issuer knows to exist. Whether or not that would be the bearer is pure speculation.

**Comment by David-Chadwick** [comment]
I agree. There are many ways of binding a VC to a subject without needing a cryptographic ID. The issuer might include the name and address and passport number of the subject in the subject's properties, which might require the holder to present their passport to the verifier at the same time as the VC.

**Comment by awoie** [comment]
I agree that we should the Verifier perspective is important to Section C. However, the meta-question is if Section C should stay in the VCDM specification or will be part of the implementation guideline.

**Comment by OR13** [comment]
I am in favor of moving informative guidance to implementation guide.

If there is normative requirements on this, I suggest we start that from scratch given the assumptions might no longer hold.
